.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+const eslintrc = require.resolve('@box/frontend/eslint/eslintrc.js');
+
+module.exports = {
+    extends: [eslintrc],
+    globals: {
+        __: false,
+        __I18N__: false,
+        ActiveXObject: false,
+        DocumentTouch: false,
+        Assert: false,
+        fixture: false,
+        PDFJS: false,
+        sinon: false,
+    },
+    rules: {
+        'class-methods-use-this': 0, // fixme
+        'import/no-cycle': 0, // fixme
```

Comment:
I have a commit to resolve this, but I'm going to PR it separately because it requires more significant code modifications.